gamemodes/terrortown/entities/weapons/weapon_ttt_binoculars.lua

```diff
@@ -46,6 +46,7 @@ SWEP.Secondary.Delay = 0.2
 SWEP.Kind = WEAPON_EQUIP2
 SWEP.CanBuy = {ROLE_DETECTIVE} -- only detectives can buy
 SWEP.WeaponID = AMMO_BINOCULARS
+SWEP.StockEquipment = true
```

Comment:
first of, i'd rename this to `builtin`

Reply:
I agree with this point because we already use this flag for other purposes, such as roles: https://github.com/TTT-2/TTT2/blob/master/lua/terrortown/entities/roles/innocent/shared.lua#L15